README.md

```diff
@@ -95,64 +97,26 @@ providing models to deserialize their responses.
 
 ## Examples
 
-```rust
-use std::{env, error::Error};
-use tokio::stream::StreamExt;
-
+```rust,no_run
 use twilight::{
-    cache::{
-        twilight_cache_inmemory::config::{InMemoryConfigBuilder, EventType},
-        InMemoryCache,
-    },
-    gateway::cluster::{config::ShardScheme, Cluster, ClusterConfig},
-    gateway::shard::Event,
+    gateway::{Cluster, ClusterConfig, Event},
     http::Client as HttpClient,
-    model::gateway::GatewayIntents,
 };
+use futures::StreamExt;
+use std::{env, error::Error};
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
     let token = env::var("DISCORD_TOKEN")?;
-
-    // This is also the default.
-    let scheme = ShardScheme::Auto;
-
-    let config = ClusterConfig::builder(&token)
-        .shard_scheme(scheme)
-        // Use intents to only listen to GUILD_MESSAGES events
-        .intents(Some(
-            GatewayIntents::GUILD_MESSAGES | GatewayIntents::DIRECT_MESSAGES,
-        ))
```

Comment:
hmm, I'd prefer if the intents section was preserved personally.

Reply:
For the most part I just used what was in the lib.rs files because I thought those would be the most up-to-date, but you're right it'd be nice to keep that.. and the project readme was probably actually more up-to-date. I'll update it to use the readme's code sample

Reply:
Should be fixed now! :)